src/bp/core/services/bot-service.ts

```diff
@@ -200,10 +200,16 @@ export class BotService {
       updatedFields.disabled = false
     }
 
-    await this.configProvider.setBotConfig(botId, {
+    const newConfig = {
       ...actualBot,
       ...updatedFields
-    })
+    } as BotConfig
+
+    if (!newConfig.languages.includes(newConfig.defaultLanguage)) {
```

Comment:
What do you think to throw that same error when we try to mount the bot and clearly flag it to the use ?